### DISCUSSION TITLE
Could fake unsubscribe sites get blocked under the Phishing blocks?

### DISCUSSION THREAD

**Discussion by dftf-stu** [uBlockOrigin/uBlock-issues#3461] [Ideas]
In uBlock Origin (the Manifest v2 version, for Firefox), I have the "Phishing URL Blocklist" enabled.

I'm surprised that as part of this filter, those fake "unsubscribe" websites, that exist purely to harvest
working e-mail addresses to then send more spam e-mails to that person, don't get blocked.

<details>
<summary>Click to see example screenshots of such sites:</summary>

![allinoursite_com](https://github.com/user-attachments/assets/494069f6-5414-432d-8517-6b490008a4f4)

![doorsstormsy_shop](https://github.com/user-attachments/assets/ba9fcd3d-fa3a-4e60-afc4-6693d688d440)

![emailservicesnetau_com](https://github.com/user-attachments/assets/c5f55f7a-8ceb-4662-81f5-2f6c71c88bc0)

![euromaillinnk_com](https://github.com/user-attachments/assets/194a6ba2-50f6-4d21-b39c-f93a9703b98d)

![findthorman_shop](https://github.com/user-attachments/assets/758200a5-6ab8-4bf2-b2bd-3f28274dc9d7)

![keydoordasher_shop](https://github.com/user-attachments/assets/255a63c5-b349-464a-8eb0-e0ca91c2b7ab)

![mybach_xyz](https://github.com/user-attachments/assets/466ed6b5-dbdc-4fe0-8059-def7ec1ac962)

![threeneptuneboot_com](https://github.com/user-attachments/assets/c620d2a0-5f32-4e62-b5a3-f3bc3c68751b)

</details>

Maybe this is something which could be considered?

I struggle to see any reason why _not_ to block them... 🤔

Thanks

**Comment by gorhill** [comment]
> I struggle to see any reason why not to block them.

Workload. We are volunteers using our free time to contribute here. We can't sanitize the whole internet. If these pages are you clicking on a link in a spam email, then the best course of action is to report these to spam email filters.

**Reply by krystian3w** [reply]
Taking `poczta.Interia.pl` as an example, their mechanisms are completely unable to learn to treat these emails as spam; it is impossible to filter by anything other than the Firebase domain (until scammers buy some new domain for $1).

In addition, companies are abandoning POP3 mechanisms for downloading external mail, as exemplified by its demise at Google and Gmailfy.

**Comment by dftf-stu** [comment]
Hi @gorhill: I have tried for years reporting the sites to both Google's _Safe Browsing_ and Microsoft's _Defender Smartscreen_ as phishing, but still they never get blocked.

I've reported this also to the _AdGuard DNS_ team, who appear to be looking to block the example domains.

Be great if uBO could also, but please close this discussion if you feel it would be out-of-scope.

**Reply by gorhill** [reply]
Closing. Whoever is free to become a contributor, the current contributors are burdened enough there is no need to pile on. If you feel strongly about this, become a contributor.

**Comment by stephenhawk8054** [comment]
At least give us the list of the URLs. We are not wizards to acknowledge every single issue on the internet, and we can't type every single URL from just the screenshots either. Everything needs the first person to report properly. If no one is bother to report properly, there's nothing to block.

**Reply by dftf-stu** [reply]
Hi @stephenhawk8054.

I'm not sure it's really-worth providing a list of domains now, given the owner of this extension has closed the ticket saying it is not something that will be looked into, but seeing as you've asked...

allinoursite.com
doorsstormsy.shop
emailservicesnetau.com
euromaillinnk.com
findthorman.shop
glaters.com
keydoordasher.shop
makitfornows.com
mybach.xyz
threeneptuneboot.com

...are all recent-examples I've seen.

Visit any of them and you'll see a "website coming soon" or "how to setup your Linux web-server" page.
To see the fake unsubscribe page, simply add either **/unsub** or **/opt-out** onto the end of those domains.
(It's usually /unsub more-often than /opt-out , but my screenshots show which.)

**Reply by stephenhawk8054** [reply]
It's closed because you just gave screenshots and asked why they are not blocked.

Issues are resolved thanks to:

1. Proper reports, and
2. Volunteers willing to spend time resolving the reports.

Without 1, there's no 2.

**Reply by dftf-stu** [reply]
@stephenhawk8054 okay, but I'm not-sure for this issue _what more_ I could report?

It seems quite simple to me: people receive a spam e-mail, they click "unsubscribe", and it then takes them to one of those sites.  Just visiting the site is enough to tell the owner "this person's e-mail address is valid", as clearly the long series of numbers and letters at the end of the URL is a tracking ID, different for every spam e-mail they send.

Whether the form on the unsubscribe pages actually works, I don't know -- I'm not a web-dev or coder, so I'm not able to tell that.

But these websites exist purely to validate whether an e-mail address is real or not, so my point was simply: why not block them, as clearly they would fall under a "phishing" category.  uBO _does_ block websites that have malware on them, such as a "your device might be infected with viruses, click here to download this tool".  So similarly, why not sites that exist purely to confirm if someone's e-mail address is valid to then send spam to?

(And yes, I have reported these sites many times to both Google's _Safe Browsing_ and Microsoft's _Defender SmartScreen_ services, but their AI clearly reviews the page and deems them not a threat, so they never block them... 🤷🏻‍♂️)

**Reply by stephenhawk8054** [reply]
As I said, at least give us the list of the URLs. If there are other steps to reproduce, you can describe more.

You want us to type every single character of the URLs from the screenshots alone?

**Reply by dftf-stu** [reply]
@stephenhawk8054: the final website doesn't have a generic page, so I can't
offer a link for that one, but for all the others hopefully this will help?

[http://allinoursite.com/unsub](http://allinoursite.com/unsub)

[http://bteux.com/unsub/](http://bteux.com/unsub/)

[http://doorsstormsy.shop/unsub](http://doorsstormsy.shop/unsub)

[http://emailservicesnetau.com/opt-out/](http://doorsstormsy.shop/unsub)

[http://euromaillinnk.com/unsub](http://euromaillinnk.com/unsub)

[http://findthorman.shop/unsub](http://findthorman.shop/unsub)

[http://glaters.com/opt-out/](http://glaters.com/opt-out/)

[http://keydoordasher.shop/unsub](http://keydoordasher.shop/unsub)

[http://makitfornows.com/opt-out/](http://makitfornows.com/opt-out/)

[http://mybach.xyz/opt-out/](http://mybach.xyz/opt-out/)

If your browser asks, you'll need to say "go to HTTP site", as none of them use HTTPS.

**Reply by dftf-stu** [reply]
@stephenhawk8054 : here's an example showing the process

![EmailUnsubExample](https://github.com/user-attachments/assets/1240a139-eb68-43a5-936c-d110b0a0ba8d)

1. Receive phishing or spam e-mail
2. Click "unsubscribe" or "opt-out" link
3. Get directed to a fake unsubscribe page, which only serves to confirm your e-mail address as active

<details><summary>Off-topic</summary>

I only reported this issue here, as I feel like these websites are "phishing", and maybe therefore they should be
blocked, to reduce the number of users who visit them, thereby reducing information spammers can harvest?

(And yes, before you say: "have users perhaps tried _not_ clicking those links?"  Sure... but users do, and the reason
we put stuff like uBO and AdGuard DNS on their devices is to protect them from themselves!  I mean, we'd likely not
need anti-virus apps if users didn't just download any random file they come-across, but, you-know... 🙄)

And given uBO (the Manifest v2 version) comes with this filter-list enabled by default...

![uBO_PhishingOption](https://github.com/user-attachments/assets/679a05c0-9fb5-4dea-ac1a-fa62a00655a6)

...I just felt these sites would come under that category? 🤔

</details>

**Reply by stephenhawk8054** [reply]
Can you share the full link of the last screenshot (the `threeneptuneboot.com`)?

**Reply by dftf-stu** [reply]
Hi @stephenhawk8054

Sure, I managed to use Bing's image-search to extract it via OCR:
[https://www.threeneptuneboot.com/o-cbgg-r56-e0f2924de59607221b2802d3e7e35cf7](https://www.threeneptuneboot.com/o-cbgg-r56-e0f2924de59607221b2802d3e7e35cf7)

Also had this new example just today:
[http://bteux.com/unsub/P2Q9MjQyOCZlaT0xNjY4MzUxMCZpZj01MTIyJmxpPTQyMg](http://bteux.com/unsub/P2Q9MjQyOCZlaT0xNjY4MzUxMCZpZj01MTIyJmxpPTQyMg)

Is it worth reporting more though -- I'm still unclear as to the status of this issue, seeing as you're asking about it, but the official project owner / maintainer closed it, suggesting it's not something the app should block, so... 🤷🏻‍♂️

**Reply by stephenhawk8054** [reply]
The issue is addressed here: https://github.com/uBlockOrigin/uAssets/commit/d8b0b01397b06330d867c2d1ce0bd5ebeb9475a3

If you see more links with the same issue, you can continue reporting in this thread.

**Reply by dftf-stu** [reply]
@stephenhawk8054: okay, thanks for adding these.  A few points though:

**[1]** Within uBlockOrigin itself, which filter-list is "badware.txt" the internal-name for?
I would assume under Built In > uBlock Filters it would be the "Badware risks" one, yes?
As such, these sites should start to be blocked by-default?  (Or does a new filter from [filterlists.com](https://filterlists.com/) have to be added first?)

**[2]** Would the blocks only occur in the Manifest v2 version of the extension, or the v3 one also?
(I'm aware the v3 one uses fewer rule-sets behind-the-scenes.)

**[3]** Looking at what you linked to, I can see the domains being blocked, but at the bottom some more-generic rules that would attempt to find such sites in future.  Are you sure these won't block actual, genuine unsubscribe pages -- like from newsletters or online shops?

Thanks

**Reply by stephenhawk8054** [reply]
1. Yeah, it's "Badware risks" list
2. They will be blocked in both MV2 and MV3 versions
3. We can't assure. If there are false positives, users can report to uBO

**Reply by dftf-stu** [reply]
Hi @stephenhawk8054.  Some more to add:

[acreplatepurple.com](https://www.acreplatepurple.com/o-gdxb-r56-32001a9acedc8833970d752d81f08336)

[alltowardsus.com](http://alltowardsus.com/unsub/)

[barck.online](http://barck.online/5tNmii4330XFGX373plqgxudbmk685RAYYFPTOUPGPYJR788FJPR1567P23)

bluecoll.com : [example 1](http://eo.bluecoll.com/opt-out/) , [example 2](http://he.bluecoll.com/opt-out/) , [example 3](http://ly.bluecoll.com/opt-out/) , [example 4](http://rf.bluecoll.com/opt-out/) , [example 5](http://zc.bluecoll.com/opt-out/)

[calmstoragefrd.shop](http://calmstoragefrd.shop/unsub/)

[canyoumore.com](http://canyoumore.com/unsub/)

[childrenoftheclouds.com](http://childrenoftheclouds.com/unsb/)

[costumercontact.com](https://costumercontact.com/opt-out/)

[dating-express.com](https://www.dating-express.com/unsubscribe/index.php)

[fechindal.com](https://www.fechindal.com/o-ztsn-r46-de300aa7c48249dfd3df2a1206067bde)

[flilisshoes.shop](http://flilisshoes.shop/unsub/)

[grandformat.pro](http://grandformat.pro/opt-out/)

[greetsld.com](http://greetsld.com/opt-out/)

[homtail.shop](http://homtail.shop/oop/14259_md/10/1711/680/25/2012)

hrdwodflors.com : [example 1](https://www.hrdwodflors.com/o-xrqr-r56-76be56cde77e05a5b05c272d9ff692df) , [example 2](https://www.hrdwodflors.com/o-lrgm-r56-9429cd466170d4ef64ae9e3c81d00ef9)

[healthcontact.info](http://healthcontact.info/unsub/)

[hunterfrosty.shop](http://hunterfrosty.shop/unsub/)

[ipvuq.gaigoivn.info](http://ipvuq.gaigoivn.info/opt-out/)

[jadimetity.com](https://www.jadimetity.com/o-wmtx-q26-99fc31577e169b0d9c9149842de98682)

**Reply by dftf-stu** [reply]
And a second batch:

[kamakurashir4.shop](http://vlfgm.kamakurashir4.shop/opt-out/)

[lapsina.us](http://www.lapsina.us/opt-out/)

[lcasinonline.com](https://lcasinonline.com/opt-out/)

[music4yyu.click](http://zgffy.music4yyu.click/opt-out)

[mwspace.io](http://mwspace.io/opt-out/)

oceanoscarkey.com : [example 1](https://www.oceanoscarkey.com/o-ngjc-r46-9d187a70a49efc722dfed1b6b8dc6c48) , [example 2](https://www.oceanoscarkey.com/o-ngjc-r46-8d8027bbacb31d4a40ac438a0824f041)

[onlinemalluk.info](http://zazla.onlinemalluk.info/opt-out/)

[pitarayly.com](https://www.pitarayly.com/o-jzbt-r56-7f85559e11077b9100577d6a1ab4e4aa)

[securedmessage.zapto.org](http://securedmessage.zapto.org/opt-out/)

[songlover.mobi](https://songlover.mobi/opt-out/)

[stepway.us.com](http://stepway.us.com/opt-out/)

[stylmd.com](http://stylmd.com/opt-out/)

[ultimatemediaevents.live](http://ultimatemediaevents.live/oop/30_md/114/6/2/3/383548)

[valuesweep.com](http://cclr.valuesweep.com/opt-out/)

[vipgrps.com](http://vipgrps.com/opt-out/)

ydns.eu : [example 1](http://de.swp.ydns.eu/opt-out/) , [example 2](http://no-red.ydns.eu/opt-out/)

**Reply by stephenhawk8054** [reply]
https://github.com/uBlockOrigin/uAssets/commit/7e979713f6b75f106163113c411160199618ce3c

https://github.com/uBlockOrigin/uAssets/commit/d6166fbe1a00b9700586c4201facfd4dceab559a

https://github.com/uBlockOrigin/uAssets/commit/694374ea9e156c5a67774a8f4ea7c1246e6e77dc

**Reply by krystian3w** [reply]
@dftf-stu https://github.com/uBlockOrigin/uAssets/discussions/32019
